config/database.pg.yml

```diff
@@ -15,7 +15,7 @@ base: &base
   adapter: postgresql
   encoding: utf8
   username: root
-  pool: 1
+  pool: 5

```

Comment:
Note, we can tackle this in a followup PR, just want to reiterate this could be a problem if rails still doesn't disconnect unused connections.  It's been a long time though so who knows.